### ISSUE TITLE
global rlimit-core

### ISSUE THREAD

**Issue by nekopsykose** [chimera-linux/dinit-chimera#21]
to get global core dumps under a user, one can `ulimit -c unlimited` somehow before starting a session or use limits.conf for their user (via pam).

dinit does not use the latter (duh) and there is no way to set the former globally from pid 1; service files have `rlimit-core =` but putting this in every service is unwieldy if one wants to 'generate a core for any crash of a system service'

the default cwd for services is /etc/dinit.d so paired with that it would be funny (not); but for me where i handle things with `kernel.core_pattern = |` it would be nice to have a way to opt-in to set the core ulimit globally starting with pid 1 so system services all dump their cores and i get to look at them later (and we would perhaps want this anyway eventually with an actual core handler). i'm not sure if this needs actual dinit changes (or if there's some way i missed) or if there's another way to make sure this is set.
